core/spec/services/spree/checkout/add_store_credit_spec.rb

```diff
@@ -58,10 +59,11 @@
     end
 
     context 'the available store credit is not enough to pay for the entire order' do
+      let(:store) { create(:store) }
       let(:expected_cc_total) { 100.0 }
       let(:store_credit_total) { order_total - expected_cc_total }
-      let(:store_credit) { create(:store_credit, amount: store_credit_total) }
-      let(:order) { create(:order, user: store_credit.user, total: order_total) }
+      let(:store_credit) { create(:store_credit, amount: store_credit_total, store: store) }
```

Comment:
@szymoniwacz let's create another store credit record here associated to another store to show that we cannot use that one